packages/app-store/utils.ts

```diff
@@ -76,7 +76,10 @@ export function getLocationGroupedOptions(integrations: ReturnType<typeof getApp
   const apps: Record<string, { label: string; value: string; disabled?: boolean; icon?: string }[]> = {};
   integrations.forEach((app) => {
     if (app.locationOption) {
-      const category = app.category;
+      // All apps that are labeled as a locationOption are video apps. Extract the secondary category if available
+      let category =
+        app.categories.length >= 2 ? app.categories.find((category) => category !== "video") : app.category;
+      if (!category) category = "video";
```

Comment:
@zomars @Udit-takkar made a change here since an app might not have the `category` field. Also added a consideration if an app belongs in another category ex. Huddle01

![image](https://user-images.githubusercontent.com/65426560/207661510-d6bf4e7f-4ae5-4d8f-8dc6-e5f844646c40.png)